### DISCUSSION TITLE
Running on mac

### DISCUSSION THREAD

**Discussion by scremnm** [Jcw87/c2-sans-fight#105] [Q&A]
I know this is a bit of an older post, but I was having some issues running the file on mac, mainly no applications to run the file (the file being .carpoj) Is there something I'm missing? if not what application do I need?

**Comment by Jcw87** [comment]
If you are only interested in playing it, you can just [click here](http://jcw87.github.io/c2-sans-fight/). You only need to care about the .caproj file if you plan on editing it. You would edit it with either [this Windows-only editor](https://www.construct.net/en/construct-2/download), or its [cross-platform sequel](https://www.construct.net/en/make-games/free-trial). I won't be accepting any pull requests that use the newer editor.
